metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -433,13 +433,7 @@ class Compilers(
         path,
         createStandaloneCompiler(
           classpath,
-          StandaloneSymbolSearch(
```

Comment:
Isn't the real issue is that we don't index JDK inside standalone symbols search? Not sure if we should allow workspace search, since it can show invalid results for auto imports etc.

Reply:
I think we would need to add `JdkSources(userConfig.javaHome)`

Reply:
Ahhhh, yea you're totally correct. I _way_ overthought this, and I just tested it with adding `JdkSources(userConfig.javaHome)`, and it works as expected. I'll make this change, and then leave out the workspace search.